index.html

```diff
@@ -1,506 +1,487 @@
+                <nav class="navbar flex">
+                    <a href="#" class="navbar__link link fz-20 fw-700 lh-20">MyLogo</a>
+                    <div class="navbar__menu">
+                        <ul class="navbar__menu__ul ul flex">
+                            <li class="navbar__menu__li">
+                                <a href="#works" class="navbar__menu__link link
+                                    fw-500 fz-15 lh-20 clr-n-600">Portfolio</a>
+                            </li>
+                            <li class="navbar__menu__li">
+                                <a href="#about" class="navbar__menu__link link
+                                    fw-500 fz-15 lh-20 clr-n-600">About</a>
+                            </li>
+                            <li class="navbar__menu__li">
+                                <a href="#contact" class="navbar__menu__link
+                                    link fw-500 fz-15 lh-20 clr-n-600">Contact</a>
+                            </li>
+                        </ul>
+                    </div>
+                    <img class="navbar__hamb" src="images/hamburguer.png"
+                        alt="hamburguer icon" />
                 </nav>
-            </div>
-        </header>
-        <!-- Headline -->
-        <section class="headline-container">
-            <div class="headline-content-container">
-                <!-- Headline Greeting -->
-                <div class="headline-greeting-container">
-                    <h1 class="headline-greeting title1">
-                        I’m Noel Foka.<br />
-                        Glad to see you!
-                    </h1>
-                </div>
-                <!-- Headline Text -->
-                <div class="headline-text-container">
-                    <p class="headline-text">
-                        I’m a software developer! I can help you build a
-                        product , feature
-                        or website Look through some of my work and
-                        experience! If you like
-                        what you see and have a project you need coded,
-                        don’t hesitate to
-                        contact me.
+            </header>
+
+            <section class="head flex">
+                <h1 class="head__title fz-40 fw-700 lh-52">
+                    I'm Noel Foka <span>Glad to see you!</span>
+                </h1>
+                <p class="head__dsc fz-16 fw-400 lh-24">
+                    I’m a software developer! I can help you build a product ,
+                    feature or
+                    website Look through some of my work and experience! If you
+                    like what
+                    you see and have a project you need coded, don’t hestiate to
+                    contact
+                    me.
+                </p>
+                <a href="#works" class="head__link link fz-16 fw-500 lh-24
+                    clr-p-400">My Projects</a>
```

Comment:
- While you made the new changes, there some new errors occurred. I am mentioning them below. Kindly fix those.
- [ ] The title is not looks like the same as the given template. It is divided into two lines in the template so it looks eye catchy.
- [ ] Also you need to use `LET'S CONNECT` but you have used `My Projects`.
![image](https://github.com/noelfoka/portfolio-Microverse-version/assets/60083437/e68e0cce-74f5-446b-bba4-a96bfde5f98b)